src/lib/scripts.js

```diff
@@ -1,52 +1,70 @@
 export const delimiters = {
-  startDelimiter: '{',
-  endDelimiter: '}'
-}
+  startDelimiter: "{",
+  endDelimiter: "}"
+};
 
-export const delimit = (text) => {
-  const { startDelimiter, endDelimiter } = delimiters
-  return `${startDelimiter}${text}${endDelimiter}`
-}
+export const delimit = text => {
+  const { startDelimiter, endDelimiter } = delimiters;
+  return `${startDelimiter}${text}${endDelimiter}`;
+};
 
 // const REQUIRED_UPLOAD_FIELDS = ['firstName', 'lastName', 'cell']
-const TOP_LEVEL_UPLOAD_FIELDS = ['firstName', 'lastName', 'cell', 'zip', 'external_id']
-const TEXTER_SCRIPT_FIELDS = ['texterFirstName', 'texterLastName']
+const TOP_LEVEL_UPLOAD_FIELDS = [
+  "firstName",
+  "lastName",
+  "cell",
+  "zip",
+  "external_id"
+];
+const TEXTER_SCRIPT_FIELDS = ["texterFirstName", "texterLastName"];
 
 // Fields that should be capitalized when a script is applied
-const CAPITALIZE_FIELDS = ['firstName', 'lastName', 'texterFirstName', 'texterLastName']
+const CAPITALIZE_FIELDS = [
+  "firstName",
+  "lastName",
+  "texterFirstName",
+  "texterLastName"
+];
 
 // TODO: This will include zipCode even if you ddin't upload it
-export const allScriptFields = (customFields) => TOP_LEVEL_UPLOAD_FIELDS.concat(TEXTER_SCRIPT_FIELDS).concat(customFields)
+export const allScriptFields = customFields =>
+  TOP_LEVEL_UPLOAD_FIELDS.concat(TEXTER_SCRIPT_FIELDS).concat(customFields);
 
-const capitalize = str => { return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase() }
+const capitalize = str => {
+  const strTrimmed = str.trim();
```

Comment:
This looks like the actual change, here.

Reply:
I will do. Thank you for the feedback.

Reply:
That is the actual change, yes.
In the future, since my linter is automatic (and for other people with automatic linting set up) I will save the file before making any changes, triggering the linter, then commit the linted changes first.